cmd/simulator/txs/agent.go

```diff
@@ -53,33 +56,70 @@ func (a issueNAgent[T]) Execute(ctx context.Context) error {
 	}
 
 	txChan := a.sequence.Chan()
+	confirmedCount := 0
+	batchI := 1
+
+	// Tracks the total amount of time waiting for issuing and confirming txs
+	var (
+		totalIssuedTime    time.Duration
+		totalConfirmedTime time.Duration
+	)
+
+	defer func() {
+		a.worker.Close(ctx)
```

Comment:
```suggestion
		_ = a.worker.Close(ctx)
```

if we are ok with ignoring the error, we should make it explicit that we're ignoring it, so that it's obvious that it was not mistakenly ignored

Reply:
thanks